README.md

```diff
@@ -113,14 +115,14 @@ In this case, the file is named `database.dart`.
 // required package imports
 import 'dart:async';
 import 'package:floor/floor.dart';
-import 'package:sqflite/sqflite.dart' as sqflite;
+import 'package:sqflite_sqlcipher/sqflite.dart' as sqflite;
 
 import 'dao/person_dao.dart';
 import 'entity/person.dart';
 
 part 'database.g.dart'; // the generated code will be there
 
-@Database(version: 1, entities: [Person])
+@Database(version: 1, entities: [Person], password: '123456')
```

Comment:
@tuuzed setting `password` in the annotation means that we have to use a constant for it. I think we should not allow this in order to avoid weak security problems.